DV Blog Post (#34)
cmu-delphi/delphi-blog: content/post/2020-10-14-dv-signal.Rmd

---

> title: "Using Electronic Health Records to Develop Real-Time COVID-19 Indicators"

Reviewer: ```suggestion
title: "A Syndromic COVID-19 Indicator Based on Insurance Claims of Outpatient Visits"
```

If "syndromic" is not familiar enough to the intended readers, we can try "Symptom-based", or drop this modifier altogether.

author: "Aaron Rumack"
date: 2020-10-14
tags: ["nowcasting", "data sources"]
output:
  html_document:
    code_folding: hide
---

> In addition to symptom surveys, one of our most prized data sources comes from health insurance claims data. Healthcare partners provide us with the counts of people going to the doctor due to COVID-like illness in counties across the entire country. We use this data to calculate a signal that estimates the level of COVID-like illness in every county where enough data is available. In this post, we discuss the data itself, what this data can tell us that the number of confirmed cases can't, and how we calculate a %CLI signal. We also investigate why the data appeared to become much less useful over time, and how we discovered that the data just needed a simple adjustment!

Reviewer: ```suggestion
In addition to symptom surveys, one of our most prized data sources is health insurance claims data. Healthcare partners provide us with daily counts of people who visited a doctor due to COVID-like illness in counties across the entire country. We use this data to calculate a signal that estimates the level of COVID-like illness in every county where enough data is available. In this post, we discuss the data itself, what this data can tell us that the number of confirmed cases can't, and how we calculate a %CLI signal. We also investigate why the data appeared to become much less useful over time, and how we discovered that the data just needed a simple adjustment!
```

<!--more-->

> In previous blog posts, we have written about indicators generated from surveys that we run through [Facebook](https://delphi.cmu.edu/blog/2020/08/26/covid-19-symptom-surveys-through-facebook/) and [Google](https://delphi.cmu.edu/blog/2020/09/18/covid-19-symptom-surveys-through-google/). In addition to these, we have also generated indicators from electronic medical records. We received access to hospital records and insurance claims data from healthcare partners covering 10-15% of the population of the United States. One indicator that we created from this data is the "Doctor Visits" signal, which estimates the percentage of office visits due to COVID-like illness.

Reviewer: Do we want to discuss the hospital records?  (1) we were not supposed to receive them based on our DUA; and (2) we no longer receive them.  The alternative is to discuss the insurance claims only.  If we want to show some results that involve the hospital records, we can add a footnote that these results are based on both insurance claims and hospital records (to which we had access until October 1).
If we decide to continue to discuss hospital records, note that they do NOT cover 10%-15% of the US population, but rather only about 40% of that 10%-15%, namely 4%-6%.

> In previous blog posts, we have written about indicators generated from surveys that we run through [Facebook](https://delphi.cmu.edu/blog/2020/08/26/covid-19-symptom-surveys-through-facebook/) and [Google](https://delphi.cmu.edu/blog/2020/09/18/covid-19-symptom-surveys-through-google/). In addition to these, we have also generated indicators from electronic medical records. We received access to hospital records and insurance claims data from healthcare partners covering 10-15% of the population of the United States. One indicator that we created from this data is the "Doctor Visits" signal, which estimates the percentage of office visits due to COVID-like illness.

Reviewer: ```suggestion
In previous blog posts, we have written about indicators generated from surveys that we run through [Facebook](https://delphi.cmu.edu/blog/2020/08/26/covid-19-symptom-surveys-through-facebook/) and [Google](https://delphi.cmu.edu/blog/2020/09/18/covid-19-symptom-surveys-through-google/). In addition to these, we have also generated indicators from electronic medical records. We received access to hospital records and insurance claims data from healthcare partners covering 10-15% of the population of the United States. One indicator that we created from the outpatient insurance claims portion of the data is the "Doctor Visits" signal, which estimates the percentage of office visits due to COVID-like illness.
```

> Electronic health records are extremely useful in providing information about epidemics in real time. A patient may interact with the healthcare system days before receiving a test (if any), and several more days before receiving a positive result and being reported as a confirmed case by a state's Department of Health. This can make our signal an **early indicator** of confirmed cases, and certainly an early indicator of deaths. Early knowledge of an increase in people with COVID-like symptoms is extremely important, as early containment can prevent an outbreak.

Reviewer: ```suggestion
Electronic health records are extremely useful in providing information about epidemics in near real time. A patient may interact with the healthcare system days before receiving a test (if any), and several more days before receiving a positive result and being reported as a confirmed case by a state's Department of Health. This can make our signal an **early indicator** of confirmed cases, and even more so an early indicator of deaths. Early knowledge of an increase in people with COVID-like symptoms is extremely important, as early containment can help contain an outbreak.
```

> Additionally, the reported case counts have other issues:

Reviewer: ```suggestion
The value of insurance claims data is even more apparent when you consider other problems with official case counts:
```

> * Limited testing capacity: Especially prevalent in the earlier days of the pandemic, limited testing capacity means that there were many people who saw a doctor for COVID-like symptoms but never even received a test. These people would be present in our data but absent from the confirmed counts.
> * Backlogs: A patient can be first reported as a confirmed case can be several days after they were tested. Since our data is linked to the date that the patient interacted with the healthcare system, we report a more accurate date.
> * Revisions: Some states have changed their definition of which tests indicate a confirmed COVID-19 case. This can be done retroactively, including hundreds or thousands of past cases and reporting them as "new" on the day on which they changed their definition. These cases are clearly assigned the wrong date, but we have no way of determining when the test was actually taken. Our data is subject to revisions due to lags in reporting, but the revisions are understood and are assigned the correct date.

Reviewer: ```suggestion
* Limited testing capacity: Especially prevalent in the earlier days of the pandemic, limited testing capacity means that there were many people who saw a doctor for COVID-like symptoms but never received a test even though they should have. These people would be present in our data but absent from the confirmed counts.
* Reporting delays:  From the moment a patient is first seen or evaluated, it may take several days (especially in the early days of the pandemic) until their specimen is collected.  It then takes a few more days for the test to be run.  Next, if the test is positive it is reported to the health authorities as required by law.  The health authorities typically do further verification and investigation (for example, determining age and other demographics) before adding the case into their public website.  As a result, many patients are reported as confirmed cases several weeks after they were first medically attended.  While insurance claims also have built-in delays (see below), a meaningful claims-based signal can be estimated within 3-4 days of the doctor's visit.
* Dating accuracy: Cumulative confirmed cases scraped and reported by sites such as JHU and USA Facts (and mirrored by COVIDcast) are based on the date the case counts were publicly posted.  As a result, the "number of new daily counts" derived by successive subtraction from these reports refers to the number of cases _published_ on that date, not those that were _lab-confirmed_ on that date.  In contrast, insurance claims records indicate the "date of service" -- the date that the patient interacted with the healthcare system, which is typically closer to the testing date.  As explained above, there can be a gap of up to several weeks between these two dates.
* Retrospective revisions: Some states have changed their definition of confirmed COVID-19 case, sometimes. This is sometime done retroactively, adding hundreds or thousands of past cases and reporting them as "new" on the day on which the definition was changed. These cases are clearly assigned the wrong date, but we have no way of determining when the test was actually taken.  Insurance claims data is not subject to such revisions.
```

> ### Doctor Visits Indicator

Reviewer: ```suggestion
### The _Doctor Visits_ (DV) Indicator
```

The Doctor Visits indicator is based solely on insurance claims. For a given geographic area and a given day, we receive the count of claims that fall into each of five categories:

> 1. Total: All claims

Reviewer: ```suggestion
1. Total: All claims, whether or not related to COVID-19.  This is also known as 'all-cause' claims.
```

> 2. Flu: ICD-10 primary code starting with J09, J10, or J11. This is an explicit diagnosis of flu.

Reviewer: ```suggestion
2. "Flu": ICD-10 primary code starting with J09, J10, or J11. This is an explicit diagnosis of flu.
```

> 3. COVID-like: ICD-10 primary code in {U07.1, U07.2, B97.29, J12.81, Z03.818, B34.2, J12.89}. Some of these codes correspond to an explicit diagnosis of COVID-19, while others indicate similar symptoms.

Reviewer: ```suggestion
3. "COVID-like": ICD-10 primary code in {U07.1, U07.2, B97.29, J12.81, Z03.818, B34.2, J12.89}. Some of these codes correspond to an explicit diagnosis of COVID-19, while others indicate similar symptoms.
```

> 4. Flu-like: ICD-10 primary code of J22 (acute lower respiratory infection) or B34.9 (viral infection, unspecified).

Reviewer: ```suggestion
4. "Flu-like": ICD-10 primary code of J22 (acute lower respiratory infection) or B34.9 (viral infection, unspecified).
```

> 5. Mixed: ICD-10 primary code of Z20.828 (suspected exposure to COVID-19) or J12.9 (viral pneumonia).

Reviewer: ```suggestion
5. "Mixed": ICD-10 primary code of Z20.828 (suspected exposure to COVID-19) or J12.9 (viral pneumonia).
```

> We calculate the percent of COVID-like illness (%CLI) as (COVID-like + Flu-like + Mixed - Flu) / Total. We subtract the Flu counts because they are unrelated to coronavirus. The indicator is available at four geographic levels: county, MSA, HRR, and state. See [here](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/doctor-visits.html) for more details.

Reviewer: ```suggestion
We estimate the percent of COVID-like illness (%CLI) as (COVID-like + Flu-like + Mixed - Flu) / Total. We subtract the Flu counts because, although they are symptomatically COVID-like, they are known to be unrelated to coronavirus, thus making the signal more directly related to COVID-19. The indicator is available at four geographic levels: county, MSA, HRR, and state. See [here](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/doctor-visits.html) for more details.
```

@mariajahja Did we ever resolved why we are subtracting 'flu'?  Given that on any date, any patient is recorded in at most one of these four categories, it would seem that we need to exclude, but not subtract, the confirmed-flu cases.  And yet maybe subtraction led to better correlations?

Reviewer 2: We originally thought to construct the signal on the "rate" scale rather than "counts". Therefore, we were subtracting the true flu rate from the mixed+flu_like rate, where the latter was thought to have counted patients that had flu but weren't definitive. But yes, the way these counts were constructed should ensure that this doesn't happen. The implementation does work on the (smoothed and padded) rate scale, rather than counts. 

Note that during the summer, the 'flu' category is essentially nil, and so this subtraction didn't do anything. Earlier on in March with a non-zero flu signal, the subtraction looked necessary to avoid having a strong corona signal that we thought was impossible. I assume that coding practices have changed since then, with doctors able to discern more clearly between covid and flu, and so for the upcoming flu season it's not clear to me whether this subtraction is helpful or necessary.

Reviewer: Thanks @mariajahja .
> the subtraction looked necessary to avoid having a strong corona signal that we thought was impossible.

This suggests to me a (admittedly post hoc) explanation:  the more confirmed-flu there is, the higher the expected fraction of flu-like that is due to flu.  If true, then rather than subtracting, we should train a sensor on these 4 covariates, and make it an adaptive sensor so it continuously adapts to the most recent coding practices.   @rumackaaron @ryantibs

> The signal is available daily starting February 1, 2020 (although understandably, the signal is nearly zero in all locations until mid-March). To preserve privacy and data integrity, we do not report a signal for a given location and date if there are less than 500 total visits in the last seven days before that date. Following these restrictions, we are able to produce estimates for about 2000 counties each day, or about two-thirds of the counties in the country and over 90% of the US population.

Reviewer: ```suggestion
The signal is available daily starting February 1, 2020 (although understandably, it is nearly zero in all locations until mid-March). To preserve privacy and data integrity, we do not report a signal for a given location and date if there are fewer than 500 total visits in the seven days ending on that date. Following these restrictions, we are able to produce each day estimates for about 2000 counties, about two-thirds of the counties in the U.S. and accounting for over 90% of the country's population.
```

                         start_day, end_day, geo_type = "state")

# For each state, average the signals over all available times
df_dv_avg = df_dv %>% group_by(geo_value) %>% summarize(value = mean(value))
df_in_avg = df_in %>% group_by(geo_value) %>% summarize(value = mean(value))

# Set a bunch of fields so that the data frames know how to plot themselves
df_dv_avg$time_value = df_in_avg$time_value = start_day
df_dv_avg$issue = df_in_avg$issue = start_day
attributes(df_dv_avg)$geo_type = attributes(df_in_avg)$geo_type = "state"
class(df_dv_avg) = c("covidcast_signal", class(df_dv_avg))
class(df_in_avg) = c("covidcast_signal", class(df_in_avg))

# Plot choropleth maps, using the covidcast plotting functionality
subtitle = paste("Averaged over", start_day, "to", end_day)
p1 = plot(df_dv_avg,
          title = "% COVID-like illness, DV signal",
          range=c(0, 20), choro_params = list(subtitle = subtitle))
p2 = plot(df_in_avg,
          title = "Daily new confirmed COVID-19 cases per 100k pop.",
          range=c(0, 30), choro_params = list(subtitle = subtitle))
grid.arrange(p1, p2, nrow = 1)
```

### Backfill

> Electronic medical records have a few interesting challenges not present in the survey data. First, is that the data is available at a significant and variable latency (we call this "backfill"). For claims related to a given day, we will receive only about half of them within the first seven days after that date. Due to backfill, we cannot make estimates of our indicator for a given day until four days have passed, and we will revise the value of that day's indicator every day.

Reviewer: ```suggestion
Medical insurance claim records have a few interesting challenges not present in the survey data. First, the data is available at a significant and variable latency, often referred to as "backfill".  Due to backfill, we cannot reliably estimate our indicator for a given date of service until four days have passed.  We continue to revise the indicator's value every day thereafter based on newly received claims.  For a given date of service, we typically receive only about half of the total claims within seven days following that date.
```

Interestingly, certain types of claims have a longer latency than others. We found that for a given date, as more claims are available, %CLI usually increases. Covid-like claims tend to have a longer latency and thus our early estimates of the indicator usually underestimate the final value.

### Weekday Effects

> Another challenge is the influence of the day of the week on %CLI. On weekends, both total counts and COVID-like counts decrease, but proportionally, total counts decrease more. This is because people are more likely to delay seeking medical care for non-acute issues than acute issues. The total counts include many visits related to non-acute issues, but almost all COVID-like counts are due to acute issues. Without correcting for this weekday effect, the %CLI curve has a "sawtooth" pattern, spiking on weekends. We derived a method to create a corrected signal that accounts for this weekday effect (for a formal mathematical description of the method, see our [API documentation](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/doctor-visits.html#day-of-week-adjustment)). Below, we see the effect of making these corrections below. When we do not correct for the weekday effect, we see a sawtooth pattern that clearly does not represent true changes in COVID-like illness within a location. However, when we make the weekday correction, we get a smooth curve that looks reasonable. It is important to note that this correction is not time-wise smoothing! Rather, we are making a point-wise correction based on the fitted $\alpha$ parameters.

Reviewer: ```suggestion
Another challenge is the influence of the day of the week on %CLI. On weekends, both total counts and COVID-like counts decrease, but proportionally, total counts decrease more. This is because doctor visits during the weekend tend to focus on acute care.  The total counts include many visits related to non-acute issues, but almost all COVID-like counts are due to acute issues. Without correcting for this weekday effect, the %CLI curve has a "sawtooth" pattern, spiking on weekends. We derived a method to create a corrected signal that accounts for this weekday effect (for a formal mathematical description of the method, see our [API documentation](https://cmu-delphi.github.io/delphi-epidata/api/covidcast-signals/doctor-visits.html#day-of-week-adjustment)). Below, we see the effect of making these corrections. When we do not correct for the weekday effect, we see a sawtooth pattern that clearly does not represent true changes in COVID-like illness within a location. However, after making the weekday correction, we get a smooth curve that looks reasonable. It is important to note that this correction is not time-wise smoothing! Rather, we are making a point-wise correction based on seven fitted parameters, one for each day of the week.
```

                                                     end_day = end_day,geo_type="state"))
data_adjusted <- suppressMessages(covidcast_signal("doctor-visits", "smoothed_adj_cli", start_day = start_day, 
                                                   end_day = end_day,geo_type="state"))

data_unadjusted = data_unadjusted %>%
  select(geo_value, signal, time_value, value) %>%
  tidyr::pivot_wider(names_from=signal,values_from = value)

data_adjusted = data_adjusted %>%
  select(geo_value, signal, time_value, value) %>%
  tidyr::pivot_wider(names_from=signal,values_from = value)

cmb_df = inner_join(data_unadjusted,data_adjusted)

cmb_df$geo_value = str_to_upper(cmb_df$geo_value)
states_to_plot = c("AZ","CA","GA","FL","NJ","NY","PA","TX","WI")
ggplot(cmb_df %>% filter(geo_value %in% states_to_plot)) +
  geom_line(aes(x=time_value,y=smoothed_cli,color="Unadjusted")) +
  geom_line(aes(x=time_value,y=smoothed_adj_cli,color="Adjusted")) +
  facet_wrap(vars(geo_value)) +
  labs(x="Time",y="%CLI",title="Doctor visit signal, with and without weekday adjustment, for nine states")
```

### Evaluating the Indicator

> We can evaluate the Doctor Visits indicator by measuring its correlation with cases. We hope that when the Doctor Visits indicator is high, cases are also high. We can measure this correlation across two axes: space and time Let $c_{it}$ and $d_{it}$ be the confirmed COVID-19 rate and DV indicator respectively, at location $i$ and time $t$. The space-wise correlation of an individual location $i$ is the rank correlation of $c_{\cdot t}$ and $d_{\cdot t}$, and the time-wise correlation of a single day $t$ is the rank correlation of $c_{i \cdot}$ and $d_{i \cdot}$. Below, we plot the distribution of time-wise correlation values for each county with at least 500 total confirmed cases. Most counties have high time-wise correlation values, indicating that when the signal is high in a certain county, the county is likely to have high case incidence.

Reviewer: Do you want to also show the average time-wise correlation over all counties with cumulative cases > N (or with population > N) as a function of N?  This will show whether the cases of poor correlation are due to small counts (less differentiates power).

```{r, message=FALSE,include=FALSE}
dv_cases_df = dv_cases_df %>%
  group_by(geo_value) %>%
  mutate(coef_6wk =
           purrr::map(time_value,
                      function(d) {
                        mask = time_value %in% (d-seq(7,41))
                        tryCatch(lm(cases[mask] ~ dv[mask])[["coefficients"]],error=function(x){NA})})) %>%
  ungroup()

dv_cases_df = dv_cases_df %>%
  mutate(slope_6wk = purrr::map_dbl(coef_6wk, function(c) { tryCatch(c[[2]], error=function(x) {NA})})) %>%
  mutate(int_6wk = purrr::map_dbl(coef_6wk, function(c) { tryCatch(c[[1]], error=function(x) {NA})})) %>%
  select(-coef_6wk)

lm_dv_cases_df = dv_cases_df %>%
  group_by(geo_value) %>%
  summarize(b=tryCatch(lm(cases ~ dv)[["coefficients"]][[1]],error=function(x){NA}),
            m=tryCatch(lm(cases ~ dv)[["coefficients"]][[2]],error=function(x){NA})) %>%
  ungroup()
```

```{r, message=FALSE,include=FALSE}
rank_cor = function(x, y) {
  return(cor(x, y, method="spearman",use="na.or.complete"))
}

> ggplot(inner_join(dv_cases_df,lm_dv_cases_df, by="geo_value") %>%

Reviewer: These plots are not showing in the HTML that @capnrefsmmat produced.

### Open Questions and Future Directions
We were happy to see that a location-specific correction was able to greatly increase the space-wise correlation. By looking at the correlations of the two corrections in the graph above, we can see that they perform quite differently early on, and perform more similarly after mid-July. This indicates that the regression coefficients of the real-time correction, which are fit using a sliding window, change significantly over time. We would like to look into this non-stationary relationship between the DV signal and case incidence to better understand why it changes, and why counties in New York and New Jersey have such a different relationship in the first place.

We would also like to see how this data compares to that of other healthcare insurance providers. Some insurance providers cover different populations with different rates, which could make their signal more or less correlated with the reported case incidence. We would expect that by combining datasets and covering more people, we can produce a more useful signal.

> We believe that this dataset will be useful for forecasting Covid cases and deaths, since a doctor visit usually precedes a confirmed test and death. We can compare time-wise and space-wise correlations for cases lagged by a week or two, as well as deaths. Forecasting models can also take the current DV signal as a feature to help predict what the case incidence will be several weeks into the future.

Reviewer: Comparing lagged correlations is simple enough that we should do it and add it to the post.

---
title: "A Syndromic COVID-19 Indicator Based on Insurance Claims of Outpatient Visits"
author: "Aaron Rumack and Roni Rosenfeld"
date: 2020-10-14
tags: ["nowcasting", "data sources"]
output:
  html_document:
    code_folding: hide
---

> In addition to symptom surveys, one of our most prized data sources is health insurance claims data. Healthcare partners provide us with daily counts of people who visited a doctor due to COVID-like illness in counties across the entire country. We use this data to calculate a signal that estimates the level of COVID-like illness in every county where enough data is available. In this post, we discuss the data itself, what this data can tell us that the number of confirmed cases can't, and how we calculate a %CLI signal. We also investigate why the data appeared to become much less useful over time, and how we discovered that the data just needed a simple adjustment!

Reviewer: ```suggestion
In addition to symptom surveys, one of our most prized data sources is health insurance claims data. Healthcare partners provide us with daily counts of people who visited a doctor due to COVID-like illness in counties across the entire country. We use this data to calculate a signal that estimates the level of COVID-like illness in every county where enough data is available. In this post, we discuss the data itself, what this data can tell us that the number of confirmed cases can't, and how we calculate a %CLI signal. We also investigate why the data appeared to become much less useful over time, and how we discovered that the data just needed a simple adjustment!  Of note, our volume of health insurance claims data is about to grow dramatically thanks to the generosity of Change Healthcare, who has donated their data to us.
```

<!--more-->

> In previous blog posts, we have written about indicators generated from surveys that we run through [Facebook](https://delphi.cmu.edu/blog/2020/08/26/covid-19-symptom-surveys-through-facebook/) and [Google](https://delphi.cmu.edu/blog/2020/09/18/covid-19-symptom-surveys-through-google/). In addition to these, we have also generated indicators from electronic medical records. We received access to hospital records and insurance claims data from healthcare partners covering 10-15% of the population of the United States. One indicator that we created from the outpatient insurance claims portion of the data is the "Doctor Visits" signal, which estimates the percentage of office visits due to COVID-like illness. We have also partnered with Change Healthcare to receive insurance claims data, and we are in the process of creating a similar signal from their data. Their dataset covers approximately 45% of the population of the United States, so we are very excited to see their "Doctor Visits" signal.

Reviewer: ```suggestion
In previous blog posts, we have written about indicators generated from surveys that we run through [Facebook](https://delphi.cmu.edu/blog/2020/08/26/covid-19-symptom-surveys-through-facebook/) and [Google](https://delphi.cmu.edu/blog/2020/09/18/covid-19-symptom-surveys-through-google/). In addition to these, we have also generated indicators from electronic medical records. We received access to hospital records and insurance claims data from healthcare partners covering 10-15% of the population of the United States. One indicator that we created from the outpatient insurance claims portion of the data is the "Doctor Visits" signal, which estimates the percentage of office visits due to COVID-like illness. We have also partnered with Change Healthcare to receive insurance claims data, and we are in the process of creating a similar signal from their data. Their dataset covers approximately 45% of the population of the United States, which we expect to significantly boost the accuracy and resolution of a "Doctor Visit" signal.
```

  geom_line(aes(time_value,corrected_cor,color="\"Real Time\" Correction")) +
  labs(x="Date",y="Correlation",title="Space-wise Correlation of DV-Corrected and Case Incidence",subtitle="Over all counties with at least 500 cumulative cases") +
  theme(legend.position = "bottom", legend.title = element_blank())
```

From the graph above, we see that correcting the DV signal by turning it into a sensor greatly improved the space-wise correlations! Furthermore, the correlation is relatively constant over time, demonstrating that the data quality did not necessarily decrease over time, since a simple linear correction stopped the deterioration.

### Open Questions and Future Directions
We were happy to see that a location-specific correction was able to greatly increase the space-wise correlation. We have seen that the regression coefficients of the correction, which are fit using a sliding window, change significantly over time. We would like to look into this non-stationary relationship between the DV signal and case incidence to better understand why it changes, and why counties in New York and New Jersey have such a different relationship in the first place.

> We would also like to see how this data compares to that of other healthcare insurance providers. As mentioned above, we have received access to data from Change Healthcare, which has a larger coverage than the provider of this data. We would be interested in comparing Change's signal to this signal in sample size, correlations, and behavior. Some insurance providers cover different populations with different rates, which could make their signal more or less correlated with the reported case incidence. We would expect that by combining datasets and covering more people, we can produce a more useful signal.

Reviewer: ```suggestion
We would also like to see how the accuracy and coverage of EMR-based signals can be increased when data coverage increases dramatically. As mentioned above, we have received access to data from Change Healthcare, which when fully extracted will increase our coverage five fold -- to over 50% of the U.S. population. We would be interested in comparing Change's signal to our current signal in sample size, correlations, and behavior. Some insurance providers cover different populations with different rates, which could make their signal more or less correlated with the reported case incidence. We would expect that by combining datasets and dramatically increasing coverage we can produce a significantly more useful signal.
```
